tests/api_components/test_recordings.py

```diff
@@ -9,6 +9,7 @@ async def test_get_name(g3: Glasses3):
     assert await g3.recordings.get_name() == "recordings"
 
 
+@pytest.mark.skip(reason="Test causes pytest to freeze")
```

Comment:
Does this test freeze even after the timeout was added?

Reply:
Yes, it would cause the test suite to hang and not continue.